tests/test_hedgehog_smoke.py

```diff
@@ -326,6 +326,255 @@ def test_pmon_smoke(setup):
     check_container_restarts_helper(duthost, container)
 
 
+def test_nat_smoke(setup):
+    duthost = setup['duthost']
+    config = setup['config']
+    build_flag = "INCLUDE_NAT"
+    service = "nat.service"
+    shell_show_nat = "show nat"
+    shell_show_nat_state = shell_show_nat + " config globalvalues | grep \"Admin Mode\" | awk '{print $4}'"
```

Comment:
Is there a problem with using `awk`?
Why we need here 2 vars and additional steps if it can be done in one? I can change name `shell_show_nat_state` to smth like `shell_get_nat_state`.
Anyway there is no CLI to check NAT state. It can be done by reading `Admin Mode` from `show nat config globalvalues`.

Reply:
> Is there a problem with using awk?

I do not mind of using 'awk'
I'm just afraid that output can be changed and then the code should be changed accordingly.
I thought that 'regex' can be more universal.

> Why we need here 2 vars and additional steps if it can be done in one? I can change name shell_show_nat_state to smth like shell_get_nat_state.

I am ok with your implementation